plugin/src/main/java/io/fabric8/maven/plugin/converter/DeploymentOpenShiftConverter.java

```diff
@@ -99,7 +101,7 @@ public HasMetadata convert(HasMetadata item) {
                 specBuilder.addNewTrigger().withType("ConfigChange").endTrigger();
 
                 // add a new image change trigger for the build stream
-                if (containerToImageMap.size() != 0) {
+                if (containerToImageMap.size() != 0 && useImageStreamTriggers) {
```

Comment:
I've no clue when an image stream trigger even makes sense - all I know is they break openshift horribly ;)